src/WireMock.Net.FluentAssertions/Assertions/WireMockAssertions.WithBody.cs

```diff
@@ -148,4 +150,20 @@ public AndConstraint<WireMockAssertions> WithBodyAsBytes(ExactObjectMatcher matc
 
         return new AndConstraint<WireMockAssertions>(this);
     }
+
+    private static string? FormatBody(object? body)
+    {
+        return body switch
+        {
+            null => null,
+            JObject jObject => jObject.ToString(Formatting.None),
```

Comment:
Since a `JObject` is a `JToken`, and as `ToString()` is not overridden, you could omit this branch.